lib/openxml/pptx/parts/presentation.rb

```diff
@@ -76,8 +76,8 @@ def add_slide_relationship(relationship)
 
         def slide_size
           OpenXml::Pptx::Elements::SlideSize.new.tap do |size|
-            size.cx = 12192000
-            size.cy = 6858000
+            size.cx = 13004800
+            size.cy = 9753600
```

Comment:
Do you want to comment those values to explain what they are?

Or: what do you think of just adding `width` and `height` getter/setter methods — that, for now, use a more human-readable unit like pixels or inches?